protobuf/plugin/deepcopy/test/deepcopypb_test.go

```diff
@@ -1027,38 +1039,23 @@ func TestOneOfCopy(t *testing.T) {
 	if !in.Equal(out) {
 		t.Fatalf("%#v != %#v", in, out)
 	}
-	if &in.Fields == &out.Fields {
```

Comment:
No. I don't think you can take the address of a function's return value, and if you could, it wouldn't yield useful information.

For example `GetField1` returns a float64. Knowing the address of that value on the stack does not tell us anything about whether a separate call to `GetField1` is returning a float64 from the same place in memory.